docs/usage.md

```diff
@@ -0,0 +1,187 @@
+# In-depth usage
+
+## Authenticate and instantiate
+
+Import the Ably JavaScript SDK and the Collaborative Spaces SDK. Create an instance of the Ably JavaScript SDK using your Ably API key and then pass that value to the Collaborative Spaces SDK constructor:
+
+```ts
+import { Realtime } from 'ably/promise';
+import Spaces from '@ably-labs/spaces';
+
+const client = new Realtime(ABLY_API_KEY);
+const spaces = new Spaces(client);
+```
+
+## Create a space
+
+A space is the virtual area of an application you want to monitor, such as a web page, or slideshow. A `space` is uniquely identified by its name. A space is created, or an existing space retrieved from the `spaces` collection by calling the `get()` method. You can only connect to one space in a single operation. The following is an example of creating a space called "demonSlideshow":
+
+```ts
+const space = spaces.get('demoSlideshow');
+```
+
+A set of `spaceOptions` can be passed to space when creating or retrieving it. `spaceOptions` enable additional properties to be set for the space. The following properties can be set:
+
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| offlineTimeout | The time in milliseconds before a member is removed from a space after they have disconnected. The default is 120,000 (2 minutes). | number |
```

Comment:
Looks like this is missing the new cursor options?

Reply:
Adding usage for cursors in a separate commit